### PR TITLE
test(cat-voices): Add running all test files

### PR DESCRIPTION
**Pull request by oldGreg5**
# Description

Have one main file with all integration tests files.

## Related Issue(s)

-

## Description of Changes

Be able to run all integration tests from all test files from 1 file by specifying '... --target=integration_test/all_test.dart'

## Breaking Changes

-

## Screenshots

-

## Related Pull Requests

-

## Please confirm the following checks

* [ ] My code follows the style guidelines of this project
* [ ] I have performed a self-review of my code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have made corresponding changes to the documentation
* [ ] My changes generate no new warnings
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] New and existing unit tests pass locally with my changes
* [ ] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: catalyst_voices/apps/voices/integration_test/app_test.dart [hunk 1]
```diff
@@ -30,96 +27,101 @@ void main() async {
...
 
-  patrolWidgetTest(
-    'Spaces drawer - guest - chooser - clicking on icons works correctly',
-    (PatrolTester $) async {
-      await $.pumpWidgetAndSettle(App(routerConfig: router));
-      await $(OverallSpacesPage.guestShortcutBtn)
-          .tap(settleTimeout: const Duration(seconds: 10));
-      await $(AppBarPage.spacesDrawerButton).waitUntilVisible().tap();
-      SpacesDrawerPage.commonElementsLookAsExpected($);
+      patrolWidgetTest(
```
Comment (dtscalac) [inline catalyst_voices/apps/voices/integration_test/app_test.dart:43]: You're sure about using patrol? It doesn't work on web

Comment (minikin) [reply catalyst_voices/apps/voices/integration_test/app_test.dart:43]: I agree with @dtscalac. Any pubs or libraries must work on Web, iOS and Android.

Comment (oldGreg5) [reply catalyst_voices/apps/voices/integration_test/app_test.dart:43]: Yes, we're using only patrol finders with pure flutter, so still running tests the same flutter way 'flutter drive ...' not 'patrol test ...'

Comment (dtscalac) [reply catalyst_voices/apps/voices/integration_test/app_test.dart:43]: Makes sense, thanks for clarification.

**Review by dtscalac — APPROVED** [review]
LGTM

**Pull request merged**